data_struct.c

```diff
@@ -0,0 +1,40 @@
+#include <stdio.h>
+#include <string.h>
+
+struct Car_type
+
+{
+    char model[30];
+    char branch[30];
+    int years;
+};
+
+struct Car_type2
+{
+    char model[30];
+    char branch[30];
+    int years;
+};
+
+int main(void)
+{
+struct Car_type Chevy;
+struct Car_type2 Ford;
+Chevy.years = 2016;
+Ford.years = 2013;
+strcpy(Chevy.branch, "Chevrolet");
+strcpy(Ford.branch, "Ford");
+strcpy(Chevy.model, "Cruze");
+strcpy(Ford.model, "Fusion");
```

Comment:
if the struct strings were 
"const char *" instead of holding the sting - you could save RAM and you would assign the pointers instead of doing. strcpy.   Keep in mind the strings you are copying already exist in ROM anyway "FORD" etc.    so you can assign them by reference.